File: Python/Operadores/Intermedio/Ejercicio_04.py
# Cobro en Estacionamiento
minutos = int(input("Ingrese la cantidad de minutos: "))
horas = int(input("Ingrese la cantidad de horas: "))
diaDeLaSemana = input("Ingrese el día de la semana: ")

montoTotal = 0

if minutos > 10:
    montoTotal = (minutos - 10) * 0.30

if horas > 0:
    montoTotal = montoTotal + horas * 0.60

if diaDeLaSemana.lower() == "martes" or diaDeLaSemana.lower() == "sábado":
    descuento = montoTotal * 0.15
    montoTotal = montoTotal - descuento

print("El monto a pagar es:", montoTotal)

File: PHP/OPERADORES/Intermedio/Ejercicio_04.php
<?php
    $minutos = 50;
    $horas = 4;
    $diaDeLaSemana = "martes";

    $montoTotal = 0;

    //Los if con expresiones condicionales
    if($minutos > 10){
        $montoTotal = ($minutos - 10) * 0.30;
    }

    if($horas > 0){
        $montoTotal = $montoTotal + $horas * 0.60;
    }

    //El == es un operador de comparación, es distinto al = que es de asignación.
    //Se basa en verificar si la variable y lo indicado son iguales, el || indica
    //que una de las dos condiciones debe cumplirse para ejecutar el código del if
    if($diaDeLaSemana == "martes" || $diaDeLaSemana == "sábado"){
        $descuento = $montoTotal * 0.15;
        $montoTotal = $montoTotal - $descuento;
    }
    
    echo "El monto a pagar es: ". $montoTotal;
?>